Gruntfile.coffee

```diff
@@ -1329,6 +1345,15 @@ module.exports = (grunt) ->
 				options:
 					testname: "Local Test - <%= grunt.template.today('yyyy-mm-dd hh:MM') %>"
 
+		"string-replace":
+			inline:
+				files:
+					'dist/wet-boew/js/': 'dist/wet-boew/js/*.js'
+				options:
+					replacements: [
```

Comment:
@lucas-hay I removed the line as requested by @LaurentGoderre :) so ready for final-final review.